libpod/container_inspect.go

```diff
@@ -890,3 +890,26 @@ func (c *Container) generateInspectContainerHostConfig(ctrSpec *spec.Spec, named
 
 	return hostConfig, nil
 }
+
+func (c *Container) hostPidNS() (bool, error) {
+	if c.config.PIDNsCtr != "" {
+		return false, nil
```

Comment:
Technically, we could go further here, and call hostPidNS() on that container - it could potentially be a pid=host container that we joined the namespace of?

I don't know if this is particularly valuable though.